masterlist.yaml

```
@@ -22720,3 +22720,14 @@ plugins:
     clean:
       - crc: 0xC64C56A5
         util: 'SSEEdit v4.0.4'
+
+  - name: 'MarkekrausCheatPowers.esl'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/58892/' ]
+    msg: [ *obsolete ]
+
+  - name: 'MarkekrausCheatPowers.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/58892/' ]
+    req: [ *SKSE ]
+    clean:
+      - crc: 0x0AABCA29
+        util: 'SSEEdit v4.0.4'
```

@markekraus Plugins on the same page are grouped together (i.e., no extra newline between them), & obsolete plugins from the same page are sorted after up-to-date plugins.

```suggestion
  - name: 'MarkekrausCheatPowers.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/58892/' ]
    req: [ *SKSE ]
    clean:
      - crc: 0x0AABCA29
        util: 'SSEEdit v4.0.4'
  - name: 'MarkekrausCheatPowers.esl'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/58892/' ]
    msg: [ *obsolete ]
```

---

Thanks!